### DISCUSSION TITLE
Invalid XML for POST_FLAT_FILE_FULFILLMENT_DATA  feed

### DISCUSSION THREAD

**Discussion by hamzaafzal9009** [amzn/selling-partner-api-models#3450] [General]
I am using the Other Carrier option and added the tracking number myself, so when the customer receives the order it shows late on the customer panel.
So, I am trying to update the order status from shipped to delivered. I haven't found any API endpoint for this task but when reading the API I noticed POST_FLAT_FILE_FULFILLMENT_DATA in feeds data so thought maybe it can update the order status still not sure though.

I am at step 7 [Download the feed processing report](https://developer-docs.amazon.com/sp-api/docs/feeds-api-v2021-06-30-use-case-guide#step-7-download-the-feed-processing-report)

And I am getting Warning The XML field heading is invalid, so values in this field were ignored. To correct this error, download the template again and use the new copy, or fix the field heading in your existing file.

Here's the XML:

``<?xml version="1.0" encoding="iso-8859-1" ?> <AmazonEnvelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="amzn-envelope.xsd"> <Header> <DocumentVersion>1.01</DocumentVersion> <MerchantIdentifier>MERCHAnT_TOKEN</MerchantIdentifier> </Header> <MessageType>OrderFulfillment</MessageType> <PurgeAndReplace>true</PurgeAndReplace> <Message> <MessageID>1</MessageID> <OrderFulfillment> <AmazonOrderID>123-1234567-1234567</AmazonOrderID> <FulfillmentDate>2022-12-13 23:20:00</FulfillmentDate> <FulfillmentData> <CarrierName>XYZ</CarrierName> <ShipperTrackingNumber>TRACKING_NUMBER</ShipperTrackingNumber> </FulfillmentData> </OrderFulfillment> </Message> </AmazonEnvelope>``

How can I fix this also am I in the right direction

**Comment by joseph74a** [comment]
I hamzaafzal9009, check the **FulfillmentDate** format:
`<FulfillmentDate>2022-12-13 23:20:00</FulfillmentDate>`
![FulfillmentDate](https://user-images.githubusercontent.com/31155146/208268440-0e9b5fcb-0987-4daf-892e-b692f43b1291.PNG)

**Comment by jheathco** [comment]
Were you able to fix this?  I'm using a date in the proper format and still having issues.
